.github/pull_request_template.md

```diff
@@ -14,9 +14,15 @@ change me!
 - [ ] CHANGELOG is updated **OR** update is not needed
 - [ ] [documentation PR](https://spaces.redhat.com/display/StackRox/Submitting+a+User+Documentation+Pull+Request) is created and is linked above **OR** is not needed
 
-### Testing
+### Quality Assurance
 
-- [ ] inspected CI results
+<!--
+General Availability requirements: https://github.com/stackrox/stackrox/blob/master/PR_GA.md
+Feature Flags usage: https://github.com/stackrox/stackrox/blob/master/pkg/features/README.md
+-->
+
+- [ ] the change is ready to be deployed in production: the feature is complete xor gated by a feature flag, and security and performance implications have been considered
```

Comment:
- the feature is complete - great boolean check
- gated by a feature flag - same
- security and performance implications have been considered - this is a bit less great... is it enough to think about that for 5 minutes?

Reply:
Refactored